app/models/community_translation.rb

```diff
@@ -10,6 +10,11 @@
 #  created_at      :datetime         not null
 #  updated_at      :datetime         not null
 #
+# Indexes
+#
+#  index_community_translations_on_community_id     (community_id)
+#  index_community_translations_on_translation_key  (translation_key)
```

Comment:
Hmm. locale is even more important. 
Maybe the best way is making index for multiple keys (community_id + locale)